src/App.js

```diff
@@ -1,25 +1,15 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { Filters } from './components/Filters';
```

Comment:
I was wondering what is the reason why you chose this name: `Filters`. ❓

Reply:
We chose the name "filters" because the buttons filter between two different paths. Is it a better way to named this kind of components?

Reply:
Thank you for your recommendations, we have already made the respective changes in this commit  b0daada